CONTRIBUTING.md

```diff
@@ -15,6 +17,8 @@ To add a new formula for `foo` version `2.3.4` from `$URL`:
 
 To contribute a fix to the `foo` formula:
 
+- You might want to check to see if someone has already contributed the same fix, since it can take time to be tested and incorporated. Look at the open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) and add the formula name to the filter box at the top of the list, so it looks like `is:pr is:open foo`. If something shows up, check to see if it accomplishes what you were about to work on. If nothing matched, then proceed.
```

Comment:
Less concerned about duplicate fixes; they don't really happen and if it's documented above then that's fine for me.